### DISCUSSION TITLE
7583 build führt zu Fehler

### DISCUSSION THREAD

**Discussion by zackmuc** [Freetz-NG/freetz-ng#1064] [Allgemein]
Beim Bauen für die 7583 bekomme ich folgenden Fehler:

> STEP 1: UNPACK
source firmware file: dl/fw/FRITZ.Box_7583-08.00.image
checking signature: valid
unpacking firmware image
unpacking filesystem image
    Filesystem on build/original/firmware/var/tmp/filesystem.image is (4:0), which is a later filesystem version than I support!
ERROR: modunsqfs: Error in build/original/firmware/var/tmp/filesystem.image

**Comment by SaMMy-lacht** [comment]
Gut das jemand den weg über Override firmware options kennt!
Aber wie soll das ganze biete funktionieren! Wenn es nicht mal Informationen hier config/.img über die 7583 im FOS8 gibt.

Denn man kann das Image erstellen mit den entsprechenden Dateien.

**Comment by zackmuc** [comment]
Und was willst du uns damit sagen?

**Comment by SaMMy-lacht** [comment]
Dass es nur 2 Möglichkeiten gibt für dich!
1 warten bis @fda77 es einbaut in freetz-ng
2 oder du dir selber einträgst und auch die dazugehörige Datei unter config/.img packst.

**Comment by LizenzFass78851** [comment]
Oder probiere es testweise unter Docker das Image für die Fritzbox zu bauen.
Ich hatte neulich einen ähnlichen fehler bei `STEP 1: UNPACK` wo auch ein Fehler auftratt der mit `ERROR: modunsqfs: ` beginnt.
Selbiges zuvor unter Docker gebaut, hat bei mir bisher immer funktioniert.

Nativ hatte ich probiert Ubuntu 24.04 und unter Docker folgenden Container `ghcr.io/freetz-ng/firmware:latest`
In beiden Fällen habe ich ein Image für die 7490 bauen wollen.

der dockerbefehl dazu wäre:
```bash
docker run -it --volume /home/testu/Dokumente/GitHub/freetz-ng:/mnt/freetz-ng --name builder --restart=always ghcr.io/freetz-ng/firmware:latest
```

der pfad zum geklonten freetz-ng repo muss ggf. angepasst werden und es muss frisch geklont sein (ohne vorher ein `make` Befehl ausgeführt zu haben.)
Dann in den Container mit `cd /mnt/freetz-ng` zum Repo gelangen und dort kann genauso mit `make` das image gebaut werden.

Falls der Rechner neugestartet wurde, kann solange der Container nicht entfernt wird, mit folgenden Befehl wieder ereicht werden ohne das Building des Images nochmal zu machen (falls bereits ein Image im Container gebaut wurde):
```bash
docker exec -it builder /usr/local/bin/entrypoint.sh
```

**Comment by SaMMy-lacht** [comment]
Ein FOS 7.5x Image ist doch ganz anders gepackt! Als das der FOS 8. Dieses ist doch schon passiert, wo es das FOS 7.8 gab. Man kann sich das auch gut bei einer 7590 mit oder ohne AX anschauen.

**Comment by fda77** [comment]
3. Möglichkeit wäre ein ganz normaler pull-request mit dem bump. Vermutlich müssen 5 Zeilen dazu geändert werden. Die github actions kümmern sich um den Rest
